beets/ui/commands.py

```diff
@@ -734,7 +734,7 @@ def choose_match(self, task):
                 search_id = manual_id(False)
                 if search_id:
                     _, _, candidates, rec = autotag.tag_album(
-                        task.items, search_id=search_id
+                        task.items, search_ids=search_id.split(' ')
```

Comment:
We should probably use `.split()` with no argument here, to split on any whitespace. That makes text entry a bit more robust to, for example, accidentally hitting the space bar twice.